### PR TITLE
Employee service and repository

### PR DESCRIPTION
**Pull request by student-google**

### CODE REVIEW

**Comment by student-google** [comment]
this closes #29

File: JobSeek/JobSeek.Api/Repository/EmploeeRepository.cs [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+        public Employee GetById(int id)
+        {
+            var employee = _dataContext.Employees
+                .Where(x => x.Id == id)
+                .FirstOrDefault();
+
+            return employee;
+        }
+        public Employee Create(Employee input)
```
Comment (zoha-shobbar) [inline JobSeek/JobSeek.Api/Repository/EmploeeRepository.cs:28]: change "input"  to "entity"

File: JobSeek/JobSeek.Api/Repository/EmploeeRepository.cs [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+            employee.Address = input.Address;
+            employee.PostalCode = input.PostalCode;
+            employee.FirstName = input.FirstName;
+            employee.LastName = input.LastName;
+            employee.SeniorityLevel = input.SeniorityLevel;
+            employee.Education = input.Education;
+            employee.BirthDate = input.BirthDate;
+            employee.Gender = input.Gender;
+            employee.MaritalStatus = input.MaritalStatus;
+            employee.MilitaryService = input.MilitaryService;
```
Comment (zoha-shobbar) [inline JobSeek/JobSeek.Api/Repository/EmploeeRepository.cs:53]: remove this lines and just write employee = entity

File: JobSeek/JobSeek.Api/Repository/EmploeeRepository.cs [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+            return employee;
+        }
+        public Employee Create(Employee input)
+        {
+            _dataContext.Employees.Add(input);
+            _dataContext.SaveChanges();
+            return input;
+        }
+
+        public Employee Update(int id, Employee input)
```
Comment (zoha-shobbar) [inline JobSeek/JobSeek.Api/Repository/EmploeeRepository.cs:35]: change "input" to  "entity"

**Review by zoha-shobbar — APPROVED** [review]

**Pull request closed without merging**